store/sqlstore/upgrade.go

```diff
@@ -195,6 +195,13 @@ func upgradeDatabase(sqlStore SqlStore, currentModelVersionString string) error
 	upgradeDatabaseToVersion529(sqlStore)
 	upgradeDatabaseToVersion530(sqlStore)
 
+	err = sqlStore.Migrate()
```

Comment:
Maybe we can check if the system is using the new versioning so that means we can skip the old migrations. What do you think?